example/android/app/src/main/java/com/example/MainActivity.java

```diff
@@ -12,4 +14,9 @@ public class MainActivity extends ReactActivity {
   protected String getMainComponentName() {
     return "example";
   }
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
```

Comment:
for PiP, let's create a `BrightcoveReactActivity` that extends `ReactActivity` and includes the necessary callbacks to support PiP. Then the `example/MainActivity` can extend that in order to support it in the react native application